modular_zubbers/code/modules/beepbooptable/table.dm

```diff
@@ -0,0 +1,46 @@
+
+
+/obj/structure/table/optable/proc/ekg(mob/living/carbon/patient)
+	var/most_severe
+	if(!patient)
+		most_severe = TRUE
+		return
+	if(patient.stat >= HARD_CRIT)
+		playsound(computer, 'modular_zubbers/sound/machines/operating_table/flatline.ogg', 5, ignore_walls = FALSE, extrarange = SHORT_RANGE_SOUND_EXTRARANGE)
+		most_severe = TRUE
+	if(patient.stat == SOFT_CRIT || BLOOD_VOLUME_SURVIVE > patient.blood_volume || organ_fatal_test(patient) && !most_severe)
+		playsound(computer, pick('modular_zubbers/sound/machines/operating_table/ekg_alert.ogg', 'modular_zubbers/sound/machines/operating_table/flatline.ogg'), 5, ignore_walls = FALSE, extrarange = SHORT_RANGE_SOUND_EXTRARANGE)
```

Comment:
My main complaint is with the flatline sound, it's not good for my ears

Reply:
https://discord.com/channels/1059199070016655462/1097484328012886067/1308227352605491213

Reply:
Likely will need to up the volume when you add it